--- checkinstallrc-dist.orig	2013-09-07 15:02:56.767005941 +0900
+++ checkinstallrc-dist	2013-09-07 15:06:45.066005843 +0900
@@ -16,14 +16,14 @@
 DEBUG=0
 
 # Location of the "installwatch" program
-INSTALLWATCH_PREFIX="/usr/local"
+INSTALLWATCH_PREFIX="/usr"
 INSTALLWATCH=${INSTALLWATCH_PREFIX}/bin/installwatch
 
 # Location of the makepkg program. "makepak" is the default, and is 
 # included with checkinstall. If you want to use Slackware's native "makepkg"
 # then set this to "makepkg"
 
-MAKEPKG=/sbin/makepkg
+MAKEPKG=
 
 # makepkg optional flags. These are recommended if running a newer Slackware
 # version: "-l y -c n"
@@ -49,7 +49,7 @@
 #   R : RPM
 #   D : Debian
 
-INSTYPE=""
+INSTYPE="R"
 
 # Storage directory for newly created packages
 # By default they will be stored at the default
@@ -108,7 +108,7 @@
 AUTODOINST=1
 
 # Are we going to use filesystem translation?
-TRANSLATE=1            
+TRANSLATE=0
 
 # Reset the owner/group of all files to root.root?
 RESET_UIDS=0               
@@ -117,7 +117,7 @@
 NEW_SLACK=1
 
 # Comma delimited list of files/directories to be ignored
-EXCLUDE=""
+EXCLUDE="/selinux"
 
 # Accept default values for all questions?
 ACCEPT_DEFAULT=0
